README.md

```diff
@@ -65,6 +65,10 @@ if you have bugs or feature requests, consider opening an issue.  If you are a d
 
 We also have a developer's mailing list, which is useful for discussing issues around the development and future of OF.
 
+## Working with CMake
+
+Using CMake will give you the best possible experience when working with openFrameworks. There is an entire page dedicated to [CMake support](CMAKE.md).
+
```

Comment:
Maybe we can say instead "CMake support is a new addition to openFrameworks. There is an entire page dedicated to [CMake support](CMAKE.md).

Reply:
I pushed a change, better?

Reply:
Great thanks!! 👍